README.md

```diff
@@ -136,8 +136,8 @@ are pure functions that the compiler can optimize. For example, the compiler
 could choose to "inline" properties and remove those that are constant or
 unchanged. In the future we hope to improve rendering time on low end devices by
 pre-processing images and text. The compiler could determine that a `Text` or an
-`Image` element is always on top of another `Image` in the same location.
-Consequently both elements could be rendered ahead of time into a single
+The `Image` element is always on top of another `Image` in the same location.
```

Comment:
That looks wrong to me, the "`Image` element ..." part is part of an entire sentence.